lib/uplink/ext/c/tests/unity.c

```diff
@@ -0,0 +1,1570 @@
+/* =========================================================================
```

Comment:
This adds quite a lot of code, which could be written using regular C.

Reply:
Are you suggesting we write our own c test assertion library!?

Reply:
I think Egon wants to say that we are using a limited number of assertions from `unity` and maybe it would be possible to avoid including it and write our assertions with regular C.

Reply:
Yeah, currently it seems a regular `==` or `memcmp` would suffice to get the same effect.

Reply:
Its resolved but no conclusion was done. I agree with Egon that currently, we are using only the simplest assertions from Unity and this code is not super necessary in our repo.

Reply:
Sorry, forgot to write the conclusion, discussed this with Bryan yesterday directly. They currently have tests and things written out using `unity.c`. Yeah, it's not necessary, however bigger priority is to get c-bindings working. We'll revisit this later once c-bindings are done.

Reply:
Thanks, I think that's the best approach.